pkg/volume/csi/csi_plugin.go

```diff
@@ -34,14 +34,14 @@ import (
 )
 
 const (
-	csiName       = "csi"
-	csiPluginName = "kubernetes.io/csi"
+	csiPluginName              = "kubernetes.io/csi"
+	csiVolAttribsAnnotationKey = "csi.kubernetes.io/volume-attributes"
```

Comment:
The node ID annotation is `csi.volume.kubernetes.io/nodeid`. Maybe make this `csi.volume.kubernetes.io/volume-attributes` to align with that?

@jsafrane thoughts?